test/ha/activator_test.go

```diff
@@ -125,7 +125,8 @@ func testActivatorHA(t *testing.T, gracePeriod *int64, slo float64) {
 		if err := pkgTest.WaitForPodDeleted(clients.KubeClient, activator.Name, system.Namespace()); err != nil {
 			t.Fatalf("Did not observe %s to actually be deleted: %v", activator.Name, err)
 		}
-		if err := pkgTest.WaitForServiceEndpoints(clients.KubeClient, resourcesScaleToZero.Revision.Name, test.ServingNamespace, test.ServingFlags.Replicas); err != nil {
+		// Check for the endpoint to appear in the activator's endpoints, since this revision may pick a subset of those endpoints.
+		if err := pkgTest.WaitForServiceEndpoints(clients.KubeClient, "activator", system.Namespace(), test.ServingFlags.Replicas); err != nil {
```

Comment:
use `activatorDeploymentName`? They should match?

Reply:
I found something better 😜